fastapi/tests/test_fastapi_demo.py

```diff
@@ -52,7 +54,7 @@ def tearDownClass(cls) -> None:
         super().tearDownClass()
 
     def _get_path(self, path) -> str:
-        return self.fastapi_demo_app.root_path + path
+        return path
```

Comment:
But this shows it is a breaking change for tests that rely on _get_app returning prefixed routes.

Reply:
But since we tests the routers and not the app in our addons, it's not a real issue.